### ISSUE TITLE
RECAP doesn't show server failures

### ISSUE THREAD

**Issue by johnhawkinson** [freelawproject/recap#210]
The server will accept documents but not process them, but it returns success to the extension.

I don't know how to fix this, and it's probably hard, but we should think about it.

It's potentially related to https://github.com/freelawproject/courtlistener/issues/769 but I don't think we want the extension polling the server for status, so maybe not.

**Comment by mlissner** [comment]
Yeah, one of my goals with the RECAP APIs is to rig them up so they can hit a configurable webhook URL (freelawproject/courtlistener#703). For a server that's using the API, that's probably the right solution to a lot of these kinds of status questions. You configure your webhook URL, and when an item is processed or fails or whatever, you get a message at your URL.

For the extensions this probably isn't the right approach...a different option is "Web Push" but it looks tricky to say the least: https://developer.mozilla.org/en-US/docs/Web/API/Push_API
